.github/workflows/ci-windows.yml

```diff
@@ -100,10 +100,12 @@ jobs:
         pip install xmipy
         pip install .
 
-    - name: Prepare for the autotests
+    - name: Install modflow executables
       working-directory: ./autotest
       run: |
-        pytest -v ci_prepare.py
+        get-modflow "C:\Users\runneradmin\.local\bin" --token $env:GITHUB_TOKEN
```

Comment:
Use [md](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/md) to ensure directory exists:
```
md C:\Users\runneradmin\.local\bin
```

Reply:
added